src/include/pbs_ifl.h

```diff
@@ -526,6 +526,7 @@ struct batch_status {
 	char		    *name;
 	struct attrl	    *attribs;
 	char		    *text;
+	struct batch_status *last;
```

Comment:
since the forward point is called next, do you want to call the backward pointer as prev? or did u really mean last?

Reply:
I meant last. This points to the last object in the current server. So that we do not have to iterate in IFL layer while aggregating. 
In fact, this is quite a waste of memory as only the head needs to have the last pointer. In our case, each batch status structure contains the last pointer. I've moved it into batch_reply structure to avoid this.

Reply:
Understood and this is fine. The list_link structure is actually very nice in this regard. You can do a GET_PRIOR(head) to get the last element, and use the same macro to go backwards if you wanted to as well. But it does not have to be done now. I will take it up later on when merging the diffstat stuff.